### ISSUE TITLE
[request] Add .sass extension support

### ISSUE THREAD

**Issue by yesh** [roots/bud#1289]
### Agreement

- [X] This is not a duplicate of an existing issue
- [X] I have read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

### Expected Behavior

bud should recognise `app.sass` entrypoint file

```
.entry({
    app: ["@scripts/app", "@styles/app"],
    editor: ['@scripts/editor', '@styles/editor'],
})
```

### Actual Behavior

bud compiler throws an error:

`Module not found: Error: Can't resolve '@styles/app'`

### Steps To Reproduce

1. install `@roots/bud-sass`
2. rename `app.css` to `app.sass`
3. run `yarn dev` or `yarn build`

### version

latest

### What package manager are you using?

yarn classic

### version

1.22.17

### Are you using pnpm?

- [ ] yes
- [ ] no

### Relevant .budfiles

[webpack.config.txt](https://github.com/roots/bud/files/8323464/webpack.config.txt)

**Comment by retlehs** [comment]
@yesh it doesn't seem like this should be expected to work from looking at other webpack projects and how sass is implemented.

**Comment by yesh** [comment]
@retlehs I think it should.
if you integrate sass, it is expected to work with both scss and sass syntax.
but the .sass extension is missing in the webpack.config generated.

with `app.scss` file the entrypoint recognizes it just fine.

**Comment by kellymears** [comment]
this is a one liner fwiw

```ts
app.hooks.on('build.resolve.extensions', e => e.add('.sass'))
```

**Comment by kellymears** [comment]
This is included in 5.7.7 🎉
